ruff_lsp/server.py

```diff
@@ -1141,6 +1159,17 @@ async def _fix_document_impl(
         extra_args=["--fix"],
         only=only,
     )
+
+    if result is None:
```

Comment:
When does `ruff` return `None`? if there were no fixes? Is this an error state (same for the other commands)

Reply:
`_run_check_on_document` will return `None` if the _extension_ chooses to ignore the file (for example, if it appears to be a stdlib file in the current environment).